pkg/config/setup/config.go

```diff
@@ -1316,6 +1316,11 @@ func InitConfig(config pkgconfigmodel.Config) {
 	// TTL refresh period represents how frequently actively detected languages are refreshed by reporting them again to the language detection handler in the cluster agent
 	config.BindEnvAndSetDefault("language_detection.reporting.refresh_period", "20m")
 
+	// config for products implemented in the client libraries
+	config.BindEnv("admission_controller.auto_instrumentation.asm.enabled", "DD_APPSEC_ENABLED_PROPAGATE")
```

Comment:
Do you think that `DD_ADMISSION_CONTROLLER_AUTO_INSTRUMENTATION_APPSEC_ENABLED` would be a better name? I think it gives a clear idea that this env var is targeting the feature in the admission controller only. (same for the other two).

Reply:
Yes, no problem. Fixed in e579273691